src/subcommands/admin/projects/list/action.ts

```diff
@@ -0,0 +1,18 @@
+import { GlobalOptions } from "@/args.ts";
+import { Table } from "cliffy-table";
+import { api } from "@/api/mod.ts";
+import { colors } from "cliffy-ansi-colors";
+
+export default async function action(
+  _options: GlobalOptions,
+) {
+  const tableData = [["name", "id"]];
+  const projects = await api.jet.listProjects();
+
+  projects.forEach(({ id, name }) => {
+    tableData.push([name, id]);
+  });
+
+  const table = Table.from(tableData);
+  api.console.log(colors.bold.underline.blue(table.toString()));
```

Comment:
不加这个颜色吧，我看着有点晃眼睛

<img width="479" alt="image" src="https://github.com/Byzanteam/jcli/assets/855678/cc4ef308-39fd-4a4f-bcf9-1d0cc583f11d">

```suggestion
  api.console.log(table.toString());
```

Reply:
这么帅的颜色🤡